package.json

```
@@ -8,6 +8,7 @@
     "@codemirror/lang-cpp": "^0.19.0",
     "@codemirror/lang-java": "^0.19.0",
     "@codemirror/lang-javascript": "^0.19.0",
+    "@codemirror/lang-php": "6.0.0",
```

```suggestion
    "@codemirror/lang-php": "^0.19.0",
```

The version number for PHP should match the other modules. When different versions are used, it results in two instances of `@codemirror/state`, which is the underlying cause of the problem